### PR TITLE
reorganize office scripts

### PR DESCRIPTION
**Pull request by szaimen**
Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

**Comment by szaimen** [comment]
I've tested onlyoffice_integrated successfully. The rest was a simple reorganization. 👍

**Comment by enoch85** [comment]
I think it's better if you explain what's going on here instead of me trying to figuring it out.

Thanks!

**Comment by szaimen** [comment]
I just try to make them better readable to use use the same logic and improve some small things while doing that.
Especially the integrated onlyoffice is now much better readable, imo!

**Comment by enoch85** [comment]
By readable, do you actually mean "possible to install both collabora and onlyoffice"?

**Comment by szaimen** [comment]
> "possible to install both collabora and onlyoffice"?

No, thats not what I intended.
If you would actualy look at the code, you would find that out ;)

**Comment by enoch85** [comment]
As far as I can see you remove Collabora instead of OnlyOffice in the Collabora script, and the opposite in the OnlyOffice script. End result = both Collabora and OnlyOffice will be installed.

Please correct me if I'm wrong.

**Comment by enoch85** [comment]
> If you would actualy look at the code, you would find that out ;) 

I looked at the code. ;)

**Comment by enoch85** [comment]
Wrong button, I will triple check. 

Need to install a test environment.

**Comment by szaimen** [comment]
> As far as I can see you remove Collabora instead of OnlyOffice in the Collabora script, and the opposite in the OnlyOffice script. End result = both Collabora and OnlyOffice will be installed.

You are wrong! Please read the code correctly!

**Comment by enoch85** [comment]
OK, so installed a fresh VPS and:

1. Installed Collabora docker (no issues)
2. Instaled Collabora integrated (no issues)
3. Installed OnlyOffice docker (Collabora app still there, but not working)
4. Installed OnlyOffice integrated (Collabora removed, OnlyOffice integrated *doesn't work*) Wrong domain

5. Uninstalled everything
6. Installed Collabora docker (no issues)
7. Installed Onlyoffcie docker (Noticed that the templates folder from Collabora is still there, other than that - all good!)
8. Installed Collabora integrated (installed but not working, maybe due to HSTS? - worked with DEMO server
9. Installed OnlyOffice Integrated (worked this time with the correct domain)

10. Installed Onlyoffice docker (no issues and domain updated correctly)
11. Installed Onlyoffice Integrated (no issues)
12. Installed Collabora docker (no issues)
13. Installed Collabora Integrated (domain value from docker still in the settings and it didn't choose the integrated option, but when I changed to use the integrated server it worked. Maybe we can force to choose the integrated version when that's installed?)
14. Installed OnlyOffice docker (tested number 3 above again, still same issue, Collabora app is still installed but not working)
15. Installed Collabora integrated (no issues)

**Errors to fix:**
1. Remove Collabora app completely when running OnlyOffice docker and integrated)
2. Remove Templates folder when uninstalling/reinstalling Collabora
3. Force to use the integrated version of Collabora when that's installed and Collabora Docker was installed before. Maybe the same for OnlyOffice?

**Comment by szaimen** [comment]
> 1. Remove Collabora app completely when running OnlyOffice docker and integrated)

Should be done now

> 2. Remove Templates folder when uninstalling/reinstalling Collabora

Thats out of scope for this PR

> 3. Force to use the integrated version of Collabora when that's installed and Collabora Docker was installed before. Maybe the same for OnlyOffice?

integrated Onlyoffice should already work, collabora now too.

File: apps/collabora_integrated.sh [hunk 1]
```diff
@@ -46,16 +46,6 @@ then
...
-    # Disable RichDocuments (Collabora App) if activated
-    if is_app_installed richdocuments
-    then
-        nextcloud_occ app:remove richdocuments
-    fi
-    # Disable OnlyOffice (Collabora App) if activated
-    if is_app_installed onlyoffice
-    then
-        nextcloud_occ app:remove onlyoffice
-    fi
```
Comment (enoch85) [inline apps/collabora_integrated.sh:39]: Why did you remove both of these?

Comment (szaimen) [reply apps/collabora_integrated.sh:39]: because they already get removed down below: https://github.com/nextcloud/vm/pull/1989/files#diff-9307a1a930d542adcec66849575564e9e6def7462586ff5a71da9c5eb985dc17R124-R135

Comment (szaimen) [reply apps/collabora_integrated.sh:39]: https://github.com/nextcloud/vm/blob/1caa815a8a2910cabfb9d3e19cb5fdff46bb6083/apps/collabora_integrated.sh#L124-L134

Comment (szaimen) [reply apps/collabora_integrated.sh:39]: ping @enoch85
